src/SmartComponents/SamplePage/SamplePage.js

```diff
@@ -23,21 +20,25 @@ class SamplePage extends Component {
     render() {
         return (
             <React.Fragment>
-                <PageHeader>
-                    <PageHeaderTitle title='Sample Insights App'/>
-                </PageHeader>
-                <Section type='content'>
+                <section className="pf-l-page__main-section pf-m-light">
+                    <div className="pf-c-content">
+                        <Title size="xxl">Sample Insights App</Title>
```

Comment:
I think we want to componentize this with PageHeader and PageHeaderTitle so that it is consistent across all Apps. We should capture the changes here in the PageHeader and PageHeaderTitle components inside of insights-frontend-components.